test/SeedData/populateDb.test.ts

```diff
@@ -0,0 +1,203 @@
+import Gateway from "../../src/models/Gateway";
+import {GatewayType} from "../../src/repositories/GatewayRepository";
+import { interacts } from '@openst/mosaic-contracts';
+import AuxiliaryChainRepositoryUtil from "../repositories/AuxiliaryChainRepository/util";
+import GatewayRepositoryUtil from "../repositories/GatewayRepository/util";
+import ContractEntityRepositoryUtil from "../repositories/ContractEntityRepository/util";
+import ContractEntity, { EntityType } from "../../src/models/ContractEntity";
+
+const Web3 = require('web3');
+
+describe('SeedData.populateDb()', () => {
+
+  let config: Config, seedData: SeedData, repositories: Repositories;
+  let web3: any;
+
+  const originChain = '12346';
+  const auxiliaryChainId = 301;
+  const zeroBn = new BigNumber('0');
+  const ostComposerAddress = '0x3c8ba8caecb60c67d69605a772ae1bb9a732fb38';
+  const ostGatewayAddress = '0x97BA58DBE58898F2B669C56496f46F638DC322d4';
+  const ostCoGatewayAddress = '0x40ce8B8EDEb678ea3aD1c9628924C903f8d04227';
+  const anchorAddress = '0xaC80704c80AB83512b48314bDfa82f79923C2Fbe';
+  const coAnchorAddress = '0xBe26124167E8a350eE806B3ba11Ddb6c8E6dc689';
+  const simpleTokenAddress = '0x325f05a75999347b7d8461BaEf274afAE0B8AE1c';
+  const ostPrimeAddress = '0x0d3E57044B1B96a257fB2ba3958c1130219A2d55';
+  
+  beforeEach(async (): Promise<void> => {
+    web3 = new Web3();
+    const eip20GatewayMockObject = {
+      methods: {
+        activated: sinon.fake.returns({call: () => Promise.resolve(true)}),
+        bounty: sinon.fake.returns({call: () => Promise.resolve(new BigNumber(10))}),
+      }
+    };
+    sinon.replace(
+      interacts,
+      "getEIP20Gateway",
+      function () {
+        return eip20GatewayMockObject as any
+      },
+    );
+    const eip20CoGatewayMockObject = {
+      methods: {
+        bounty: sinon.fake.returns({call: () => Promise.resolve(new BigNumber(10))}),
+      }
+    };
+    sinon.replace(
+      interacts,
+      "getEIP20CoGateway",
+      function () {
+        return eip20CoGatewayMockObject as any
+      },
+    );
+    const mosaicConfigPath = 'test/Facilitator/testdata/mosaic-config.json';
+    const facilitatorConfigPath = 'test/FacilitatorConfig/testdata/facilitator-config.json';
+    config = Config.fromFile(mosaicConfigPath, facilitatorConfigPath);
+    sinon.replaceGetter(
+      config,
+      'originWeb3',
+      function () {
+        return web3;
+      }
+    );
+    sinon.replaceGetter(
+      config,
+      'auxiliaryWeb3',
+      function () {
+        return web3;
+      }
+    );
+    repositories = await Repositories.create();
+    seedData = new SeedData(
+      config,
+      repositories.gatewayRepository,
+      repositories.auxiliaryChainRepository,
+      repositories.contractEntityRepository
+    );
+    await seedData.populateDb();
+  });
+
+  it('should create entry in auxiliary_chains table', async () => {
+    const auxiliaryChain = new AuxiliaryChain(
```

Comment:
We can refactor the verification code into a different methods.
e.g. verifyGatewayData, verifyAuxiliaryChainData etc.
We can have one it and the verification methods can be called from them.
Also below code should be part of `it`
```
await seedData.populateDb();
```